docs/quick-start.md

```diff
@@ -298,8 +298,17 @@ values to match against, and can contain multiple variants.
 
 ## Local Declarations
 
-TODO
+A **local** declaration binds a variable to the value of an expression. `.local` is like `const` in JavaScript. Think of it as a convenient way to reuse the result of an operation without repeating it.
```

Comment:
After the first example we can mention that this is particularly useful for `.match` if you want to use the same formatting across multiple variants of a match statement.